services/checks/checks.go

```diff
@@ -451,8 +452,36 @@ func (s *Service) minPMMAgentVersion(t check.Type) *version.Parsed {
 	}
 }
 
-// executeChecks runs all available checks for all reachable services.
-func (s *Service) executeChecks(ctx context.Context) error {
+func (s *Service) filterChecks(checks []check.Check, disable, enable []string) []check.Check {
+	var res []check.Check
+	dm := make(map[string]struct{}, len(disable))
+	for _, e := range disable {
+		dm[e] = struct{}{}
```

Comment:
IMO `dm` and `em` looks confusing, maybe specify full name e.g `disabledMap`?

Reply:
Renamed